### ISSUE TITLE
tlapse does not take/save screenshots

### ISSUE THREAD

**Issue by jkulak** [typicode/tlapse#5]
Hi, great idea, I would love to use it, but:

`/app # tlapse -- localhost:3000`

results in:

```
Screenshots directory: ./tlapse
Interval: 1m
01:21:14
01:22:15
```

but the content of the directory stays empty.

I run `wget localhost:3000` and it returns an index.html that I also see in the browser.

Changing the directory, resolution, interval didn't change anything.

I'm using it inside a Docker container `FROM mhart/alpine-node:7.4`.

I am running it as root, and did the `npm install -g tlapse` as root.

**Comment by jkulak** [comment]
After running tlapse with changes from my pull request (https://github.com/typicode/tlapse/pull/6), I see

```
eb4dbced958c:/app/tlapse/tlapse# node index.js -d ./screens -e 1s -- localhost:3000
Screenshots directory: ./screens
Interval: 1s
Error
stdout:
stderr: events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn /app/tlapse/tlapse/node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs ENOENT
    at exports._errnoException (util.js:1022:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:359:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
    at Module.runMain (module.js:607:11)
    at run (bootstrap_node.js:420:7)
    at startup (bootstrap_node.js:139:9)
    at bootstrap_node.js:535:3
^C
```

**Comment by typicode** [comment]
Hi @jkulak,

Just published a new version with your PR (`v0.1.7`), should be better.
Thanks again :)
